manifests/ssltools/nssdb.pp

```diff
@@ -1,25 +1,15 @@
 # Sets up nssdb
 class certs::ssltools::nssdb (
-  $nss_db_dir = $certs::nss_db_dir,
```

Comment:
If we drop this we break the interface. I'd opt to keep it and introduce the `pki_dir` parameter so it's an additive change.

Reply:
I think the broken interface shouldn't actually break anything (except possibly tests, which I didn't check).

Since in manifests/qpid.pp we `include certs::ssltools::nssdb`

Reply:
Technically, for our use case that is true, but this is not marked as a private class. So you would be breaking a public ally available interface.

Reply:
I think within the context of the rather sweeping changes we are making in this module, labeling this as a breaking change and releasing it all under a new major version would be OK. What do you think?

Reply:
I think we should aim to be least invasive as possible. You'll have to additionally go clean up https://github.com/theforeman/puppet-certs/blob/master/manifests/init.pp#L104 as you are now defining this in two locations.

From an interface definition standpoint, I think declaring what the nssdb path is makes more sense than declaring a parent folder and then always naming it nssdb. But I am getting into broader design thinking versus how we use this today and how we want to use this tomorrow (e.g. if we move to the planned model of puppet-qpid declaring the nssdb we would likely want to put it in /etc/qpid/certs for example)